schema/deploy/warehouse/site/data.sql

```diff
@@ -5,44 +5,113 @@ begin;
 
 insert into warehouse.site(identifier, details)
     values
+        ('Apple',                                   '{"category": "community",  "type": "apple-study", 
+            "sample_origin": "apple"}'),
+        ('AvalonFederalWayCareCenter',              '{"category": "community",  "type": "shelter", 
+            "swab_site": "avalon"}'),
         ('BirchTreeAcademy',                        '{"category": "community",  "type": "childcare"}'),
+        ('BoeingField',                             '{"category": "community",  "type": "shelter", 
+            "swab_site": "boeing"}'),
+        ('BurienNursingAndRehab',                   '{"category": "community",  "type": "nursingHome", 
+            "swab_site": "burien_nursing"}'),
         ('CapitolHillLightRailStation',             '{"category": "community",  "type": "publicSpace",
             "swab_site": "%cap(it[ao]l)?_hill%"}'),
         ('ChildcareCenter70thAndSandPoint',         '{"category": "community",  "type": "childcare",
             "swab_site": "cc_sand_point"}'),
-        ('ChildcareSwabNSend',                      '{"category": "community",  "type": "childcare-study"}'),
+        ('ChildcareSwabNSend',                      '{"category": "community",  "type": "childcare-study",
+            "sample_origin": "childcare"}'),
         ('ChildrensHospitalBellevue',               '{"category": "clinic",     "type": "clinic"}'),
         ('ChildrensHospitalSeattle',                '{"category": "clinic",     "type": "clinic",
             "swab_site": "sch_ed"}'),
         ('ChildrensHospitalSeattleOutpatientClinic','{"category": "clinic",     "type": "clinic",
             "swab_site": "ballard_clinic|community_clinic"}'),
         ('ChildrensSeaMar',                         '{"category": "clinic",     "type": "clinic"}'),
+        ('ClementPlace',                            '{"category": "community",     "type": "shelter",
+            "swab_site": "clement"}'),
+        ('ClinicalAdultFamilyHomes',                '{"category": "clinic",     "type": "adultFamilyHome",
+            "sample_origin": "clinical_afh"}'),
+        ('ClinicalWorkplace',                       '{"category": "clinic",     "type": "clincalWorkplace",
```

Comment:
@kschwabefry sounds good. Would this apply only to ClinicalWorkplace or ClinicalAdultFamilyHomes as well? I can't seem to remember if the study test only the residents at the AFH and/or also the workers?

Reply:
Discussed at standup, but recording here as well: AFH may have tested both residents and staff (similar to childcare studies testing children and staff). The boundary is a little fuzzy but since the main testing is with residents, we'll classify AFH as a SNF rather than a workplace.